data/signin.js

```diff
@@ -0,0 +1,265 @@
+class Signin{
+
+    constructor() {
+        this.selector_type = 'dusk';
+
+        this.params = {
+            group:{
+                count: null,
+                name: null,
+            }
+        }
+
+        this.selectors = {
+            email: 'signin-email_or_username',
+            password: 'signin-password',
+            submit: 'signin-signin',
+            page_heading: 'h3',
+            home_page_heading: 'h2',
+            page_subheading: 'p',
+            alert_box: 'alertdialog',
+            menu_item: 'menuitem',
+            logout_button: '=Logout',
+            eye_icon: 'eye',
+            forgot_password: '=Forgot Password?',
+            link_text: '=VaahCMS'
+        }
+
+        this.params.page = {
+            id: "SL",
```

Comment:
@satyajeet-s001 why this is `SL` instead of `SI`?

Reply:
Updated sir!.